### ISSUE TITLE
[css-animations] Interaction of display:none parent with svg:use.

### ISSUE THREAD

**Issue by george-steel** [w3c/csswg-drafts#4413]
Should CSS animations be canceled/not started on an element with a display:none ancestor which is made visible through a svg:use element. Currently Chrome will cancel animations if in this case while Firefox will continue/start animations as long as the element is visible.

[Test case](https://codepen.io/george-steel/pen/xxxwqPZ)

**Comment by emilio** [comment]
This is a bigger issue than that, AIUI.

Chrome doesn't implement the SVG 2 styling rules at all, so what gets styled is the element in the display: none subtree, not the element in the `<svg:use>` shadow tree (which is the most reasonable thing in this case, I think).

cc @dirkschulze @AmeliaBR

**Comment by george-steel** [comment]
You're right. This goes a lot deeper than I thought at first. The spec seems much more sensible then blink's current implementation.

**Comment by emilio** [comment]
You should "just" fix [issue 785246](https://bugs.chromium.org/p/chromium/issues/detail?id=785246) then :)

I even had a WIP at some point for that...

**Comment by george-steel** [comment]
Looking through the relevant code, that is probably a good idea.
